alpine-38.dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# apt-utils seems missing and warnings are shown, so we install it.
+RUN apk update && \ 
+ echo 'UTC' > /etc/timezone && \
+ apk add tzdata file bash && \
```

Comment:
Also make sure `sudo` is installed, and locales? Are they already installed?

Reply:
Added sudo, but unfortunately locales package is not available on Alpine there is one workaround available as of now https://github.com/gliderlabs/docker-alpine/issues/144#issuecomment-339906345 , should I add these in base image? I would suggest that any apps depending on locale should take care of it .

Reply:
OK, if Alpine does not have locales, then this is OK.